Files/ViewModels/ItemViewModel.cs

```diff
@@ -1072,9 +1072,10 @@ public async Task LoadExtendedItemProperties(ListedItem item, uint thumbnailSize
         private async Task<ImageSource> GetItemTypeGroupIcon(ListedItem item, BaseStorageFile matchingStorageItem = null)
         {
             ImageSource groupImage = null;
-            if (item.PrimaryItemAttribute != StorageItemTypes.Folder)
+            if (item.PrimaryItemAttribute != StorageItemTypes.Folder || item.IsZipItem)
             {
-                var headerIconInfo = await FileThumbnailHelper.LoadIconFromPathAsync(item.ItemPath, 76, ThumbnailMode.ListView);
+                var headerIconInfo = await FileThumbnailHelper.LoadIconWithoutOverlayAsync(item.ItemPath, 64u);
```

Comment:
I understand there's no way to force a StorageFile to return the icon instead of the thumbnail, correct? Does `ThumbnailMode.DocumentsView` make any difference?

Reply:
doesn't seem like it